src/components/distributions/editor/TextForm/TextInput.js

```diff
@@ -33,6 +33,16 @@ class TextInputEditor extends Component {
     return this.setState({editorState})
   }
 
+  handleReturn(e) {
+    const shifted = e.shiftKey
```

Comment:
I'm not sure naming this 'shifted' adds any meaningful clarity; I'd remove it and just test on e.shiftKey directly.